yarn.lock

```diff
@@ -4,154 +4,100 @@
 
 "@algolia/autocomplete-core@1.17.9":
   version "1.17.9"
-  resolved "https://registry.yarnpkg.com/@algolia/autocomplete-core/-/autocomplete-core-1.17.9.tgz#83374c47dc72482aa45d6b953e89377047f0dcdc"
+  resolved "https://registry.npmjs.org/@algolia/autocomplete-core/-/autocomplete-core-1.17.9.tgz"
```

Comment:
What is the reason to change `yarn.lock`?

Reply:
yarn | npm are black magic and I've overlooked unnecessary change when was preparing PR. 

Now I removed yarn.lock changes from PR.